app/code/VConnect/Blog/Block/Widget/PostsList.php

```diff
@@ -52,17 +51,16 @@ public function getPostUrl(Post $post): string
     }
 
     /**
-     * @return \VConnect\Blog\Model\ResourceModel\Post\Collection
+     * @return array
      */
-    private function getPostsCollection(): Collection
+    private function getPostsCollection(): array
```

Comment:
logically it must be named getPosts or getPostItems. It doesn't return collection object. Kinda substitution of concepts